ui/src/org.ts

```diff
@@ -42,6 +42,8 @@ export const getIdAvailability = (id: string): Promise<boolean> =>
   getOrg(id).then(org => !org);
 const validateUserExistence = (handle: string): Promise<boolean> =>
   user.get(handle).then(user => !!user);
+const validateNewMember = (orgId: string) => (handle: string): Promise<boolean> =>
+  getOrg(orgId).then(org => org.members.every((member) => member.handle != handle));
```

Comment:
this validates a user by checking that all the members of the involved org have a different handle. I'd expect it to, instead, validate that it doesn't find any member with that handle. So instead of `every`, I'd expect a `! any` (pseudo-code here).

Reply:
@NunoAlexandre I agree that it would read better, but as far as I could see, the javascript [Array class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) does not provide `any`.

Reply:
it seems like [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) would cut it!

Reply:
changed in 55a05fbd